packages/volto/src/components/manage/Widgets/RadioGroupWidget.jsx

```diff
@@ -0,0 +1,210 @@
+/**
+ * RadioGroupWidget component.
+ * @module components/manage/Widgets/RadioGroupWidget
+ */
+
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { compose } from 'redux';
+import filter from 'lodash/filter';
+import map from 'lodash/map';
+import { injectIntl } from 'react-intl';
+import {
+  getVocabFromHint,
+  getVocabFromField,
+  getVocabFromItems,
+} from '@plone/volto/helpers/Vocabularies/Vocabularies.js';
+import { Radio } from 'semantic-ui-react';
+import FormFieldWrapper from '@plone/volto/components/manage/Widgets/FormFieldWrapper';
+import {
+  getVocabulary,
+  getVocabularyTokenTitle,
+} from '@plone/volto/actions/vocabularies/vocabularies';
+
+/**
+ * RadioGroupWidget component class.
+ * @function RadioGroupWidget
+ * @returns {string} Markup of the component.
+ */
+class RadioGroupWidget extends Component {
```

Comment:
It would be nice to add new components as function-style components so we don't have to rewrite them later.